config/generateRunFile.coffee

```diff
@@ -457,105 +457,64 @@ generateDev = (KONFIG, options) ->
       command -v kubectl          >/dev/null 2>&1 || { echo >&2 "I require kubectl. To install kubectl: \
               (curl -L -O https://storage.googleapis.com/kubernetes-release/release/v1.6.4/bin/$(uname | awk '{print tolower($0)}')/amd64/kubectl && chmod +x kubectl && mv kubectl /usr/local/bin/)"; exit 1;}
 
-      # for local development, minikube must be deleted and recreated if a disk resize is needed
       export CHANGE_MINIKUBE_NONE_USER=true
       sudo -E minikube start --vm-driver=none
 
       sleep 60
 
+      envsubst < deployment/kubernetes/build-pod.yaml.template > deployment/kubernetes/build-pod.yaml
```

Comment:
I missed this on previous PR. Could you use (or eliminate the necessity) something else other than `envsubst` MacOS does not come with it.

Reply:
I think we can make it a required dependency. It is a part of `gettext`, actually. So, it might not be installed by GNU+Linux distros. What do you think?

```
brew install gettext
brew link --force gettext
```

It is located in $BREW_PREFIX/opt/gettext/bin/envsubst.

Reply:
We might even not need this if we start working generating templates from JSON objects [related](https://github.com/koding/koding/pull/11419#discussion_r136932356)